.github/workflows/test_on_push.yml

```diff
@@ -154,6 +154,9 @@ jobs:
         with:
           enable-cache: true
           cache-dependency-glob: "**/requirements*.txt"
+      - name: Install FreeTDS (macOS)
```

Comment:
Hey @ZoranPandovski,
Is it OK to do this since it assumes that environments of users running MacOS will have FreeTDS installed?